src/main/java/ro/msg/learning/shop/controllers/ProductController.java

```diff
@@ -0,0 +1,48 @@
+package ro.msg.learning.shop.controllers;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import ro.msg.learning.shop.domain.DTOs.ProductDTO;
+import ro.msg.learning.shop.mappers.ProductMapper;
+import ro.msg.learning.shop.services.ProductService;
+
+import java.util.List;
+
+@RestController
+@RequestMapping("/products")
+public class ProductController {
+    private final ProductService productService;
+    private final ProductMapper productMapper;
+
+    @Autowired
+    public ProductController(ProductService productService, ProductMapper productMapper) {
+        this.productService = productService;
+        this.productMapper = productMapper;
+    }
+
+    @PostMapping(value = "")
+    public void create(@RequestBody ProductDTO productDTO) {
+        productService.create(productMapper.toProduct(productDTO));
```

Comment:
its generally a good practice to return the saved/created value as dto or at least a Response Entity with a status. Your controller may be a layer in another app and it might need to "know" what happened to the dto it just attempted to save